jetty-util/src/main/java/org/eclipse/jetty/util/resource/Resource.java

```diff
@@ -921,4 +913,97 @@ public static URL toURL(File file) throws MalformedURLException
     {
         return file.toURI().toURL();
     }
+
+    /**
+     * Parse a list of String delimited resources and
+     * return the List of Resources instances it represents.
+     * <p>
+     * Supports glob references that end in {@code /*} or {@code \*}.
+     * Glob references will only iterate through the level specified and will not traverse
+     * found directories within the glob reference.
+     * </p>
+     *
+     * @param delimitedReferences the comma {@code ,} or semicolon {@code ;} delimited
+     * String of resource references.
+     * @param globDirs true if glob references return directories within the glob as well
+     * @return the list of resources parsed from input string.
+     */
+    public static List<Resource> fromList(String delimitedReferences, boolean globDirs) throws IOException
```

Comment:
can't say I love `delimitedReferences`.
How about `resourceList`

Reply:
I'm not sold.
To me, that parameter is not a list, and it is not resources.

It's a String.
It supports:
* 1 or more entries. (using simple delimited behaviors with comma or semicolon).
* each entry can be one of the following types:
  * a classpath reference.
  * It can be a relative path (to the working directory).
  * It can be a fully qualified path reference.
  * It can be a Java URI. (no side effects)
  * It can be a Java URL. (with side effects)
  * It can be a path reference with a simple glob suffix.
* no entry can, itself, have an unencoded comma `,` or semicolon `;` in its name.

Reply:
How can that arg not be a list of resources?  The method is  `fromList` not `fromStringWithDelimitersSeparatingThingsThatCanBeVariouslyResolvedToResources`?

How about:
```
List<Resource> fromString(...)
```